### DISCUSSION TITLE
Project Table View is too compact and difficult to find specific issues

### DISCUSSION THREAD

**Discussion by mikefactorial** [community/community#11567] [Projects and Issues]
Been working with the beta projects for a while and I find it difficult to find specific issues in a list at a glance. The standard issues view has the title of the issue in a heavier weight and larger font with the 16px vs 14px and the padding around the issues makes it easier to read. Would be great to reproduce the standard issues view in projects and still include some of the details under the title including Issue Number etc.

**Comment by vmathkar** [comment]
Examples of behavior that contributes to creating a positive environment include:

Using welcoming and inclusive language
Being respectful of differing viewpoints and experiences
Gracefully accepting constructive criticism
Focusing on what is best for the community
Showing empathy towards other community members
